### ISSUE TITLE
[BUG] Calculated styles for Slider widget in BokehJS are inconsistent with Bokeh in Python

### ISSUE THREAD

**Issue by ndmlny-qs** [bokeh/bokeh#12109]
### Software versions

Python version      :  3.8.0 | packaged by conda-forge | (default, Nov 22 2019, 19:11:38)
IPython version     :  8.2.0
Tornado version     :  6.1
Bokeh version       :  2.4.2
BokehJS static path :  miniconda3/envs/bokeh-docusaurus/lib/python3.8/site-packages/bokeh/server/static
node.js version     :  v17.9.0
npm version         :  8.5.5
Operating system    :  Linux-5.16.11-1-MANJARO-x86_64-with-glibc2.10

### Browser name and version

latest chrome, brave, opera

### Jupyter notebook / Jupyter Lab version

jupyterlab 3.3.4

### Expected behavior

Slider handle follows the grey bar as seen in a Jupyter notebook. Note that the `transform` style and compare with the JS version below.

![python-slider](https://user-images.githubusercontent.com/60523020/165633410-4428acc9-3c7a-44c7-81e2-e807c1ac8601.gif)

### Observed behavior

The `transform` style for the slider using BokehJS is off by a factor of 10 in the translation. When I adjust the translation by a factor of 10 in the console, the slider handle correctly aligns itself with the grey bar on the slider. If I move it again, the calculated transform value takes over and jumps the handle to a new position.

![slider](https://user-images.githubusercontent.com/60523020/165630533-1a905b80-c565-4179-8aa2-e8f2300a0c90.gif)

### Example code

```Python
// I'm still learning how to make MREs for JS, so apologies in advance.

import React from "react";
import * as Bokeh from "@bokeh/bokehjs";

export const BKSlider = () => {
  return (
    const bwSlider = Bokeh.Widgets.Slider({
      start: 0.01,
      end: 2.0,
      value: 1.0,
      step: 0.01,
      width: 1000,
    });
    Bokeh.Plotting.show(bwSlider);
  )
}
```

**Comment by bryevdv** [comment]
@ndmlny-qs Can you try this with a 3.0 dev release? There's been alot of DOM work in `branch-3.0` so it is possible this is already resolved there.

**Comment by ndmlny-qs** [comment]
Closing as @bryevdv you are correct. This has been fixed in the 3.0 build. I will make the transition to 3.0 now as I saw a few API changes. Thanks again!

![bokehjs-tool1](https://user-images.githubusercontent.com/60523020/166127258-148d6d68-10f8-4d68-af42-71e3faf0a171.gif)
